c++ training/ArraysVectors/Arrays-Vectors/ArraysVectors/DynamicArray.h

```diff
@@ -0,0 +1,241 @@
+#pragma once
+
+template<typename T>
+class TDynamicArray
+{
+private:
+	int Size = 0;
+	int Capacity = 1;
+	T* Data = new T[Capacity];
```

Comment:
Yup, I think I will change it... However I don't think it should have that last problem just because my Append will set the new size to the sum of the two lists sizes, not their capacities